SPECS-EXTENDED/frr/frr.spec

```diff
@@ -54,10 +54,8 @@ BuildRequires:  readline-devel
 BuildRequires:  systemd-devel
 BuildRequires:  systemd-rpm-macros
 BuildRequires:  texinfo
-%if %{with_check}
 BuildRequires:  python3-pip
 BuildRequires:  python3-pytest
-%endif
```

Comment:
This change was probably required to build outside of our toolkit, but since we define this macro in our tooling, we can revert this change.